### ISSUE TITLE
<Advanced HTML and CSS>: <Inconsistent link leads to download instead of opening in browser>

### ISSUE THREAD

**Issue by leah205** [TheOdinProject/curriculum#28796]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

In the Project: homepage section both the tablet and mobile design files open in a new tab while the desktop file is downloaded.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-advanced-html-and-css-homepage

**Comment by Rajaravi99** [comment]
Yes, i checked looks like what you are saying is right. This can be worked upon.

**Comment by mashazyu** [comment]
@Rajaravi99 , hello!

What is the correct behavior - downloading or opening in the separate tab? And I would like to work on this issue, if it's up for grabs.

Kind regards,
Maria

**Comment by Rajaravi99** [comment]
@mashazyu , Hello
I agree, there should be singularity in how various links open on any webapp. According to me the correct behavior should be opening in a separate tab. But lets wait for, what the project owners have to say about this.

**Comment by MaoShizhong** [comment]
Thanks for reporting this. It seems the larger file size for the desktop image is what's causing our CDN to serve it as a downloadable stream instead of just an image like with the other two. This can be resolved by replacing the desktop image and link with a compressed version (smaller file size).

## Acceptance criteria

This is a 2 PR process as per our [instructions for adding/replacing curriculum images](https://github.com/TheOdinProject/curriculum/wiki/Adding-Images-to-the-Curriculum).

- [ ] The desktop image is replaced with a smaller file size version (provided below).
- [ ] The relevant link href in the project spec is replaced with the new image's Statically link.

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

Compressed desktop image:

<img width="1440" alt="portfolio desktop" src="https://github.com/user-attachments/assets/3df7ab0f-2ed7-42aa-ad1f-a85afdd043b0">

**Comment by Rajaravi99** [comment]
Hi, I am happy to work on it. You can assign it to me.
